server_status/views.py

```diff
@@ -69,12 +68,14 @@ def get_redis_info():
         ConnectionError as RedisConnectionError,
         ResponseError as RedisResponseError,
     )
-    try:
-        url = settings.BROKER_URL
-        _, host, port, _, password, database, _ = parse_redis_url(url)
-    except AttributeError as ex:
-        log.error("No redis connection info found in settings. Error: %s", ex)
+    for conf_name in ('CELERY_BROKER_URL', 'BROKER_URL', 'REDIS_URL', ):
+        if hasattr(settings, conf_name):
+            url = getattr(settings, conf_name)
+            break
+    else:
+        log.error("No redis connection info found in settings.")
         return {"status": NO_CONFIG}
+    _, host, port, _, password, database, _ = parse_redis_url(url)
 
```

Comment:
This is hardcoding the assumption that the Celery broker will only ever be Redis.